lib/exercism.rb

```diff
@@ -43,6 +43,8 @@
 require 'exercism/log_entry'
 require 'exercism/language_track'
 require 'exercism/user_lookup'
+require 'exercism/db_source'
+require 'exercism/github_source'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.